### PR TITLE
#159237274 Configure Babel

### PR DESCRIPTION
**Pull request by ascii-dev**
#### What does this PR do?
Configure Babel

#### Description of Task to be completed?
- Install babel-cli and babel-preset0-env
- Configure package.json to run babel before `npm start`
- Add babel config file

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#159237274](https://www.pivotaltracker.com/story/show/159237274)

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
+}
```
Comment (anuonifade) [inline .babelrc]: New line

File: .gitignore [hunk 2]
```diff
@@ -34,3 +34,4 @@ node_modules
 # Optional REPL history
 .node_repl_history
 
+dist/
```
Comment (anuonifade) [inline .gitignore]: New line

File: .babelrc (hunk 1 above)
Comment (ascii-dev) [reply .babelrc]: Done

File: .gitignore (hunk 2 above)
Comment (ascii-dev) [reply .gitignore]: Done

**Pull request merged**